packages/app-transfer/src/Fees.tsx

```diff
@@ -80,7 +80,7 @@ class FeeDisplay extends React.PureComponent<Props, State> {
     );
 
     let txfees = fees.baseFee
-        .add(fees.transferFee)
+        .add(fees.transferFee ? fees.transferFee : new Balance(0))
```

Comment:
Not sure why this is in this PR. And not sure what this is supposed to fix since it should be there. However, if this is an issue it means the `= ZERO_FEES` does not do the trick. (And the issue is actually elsewhere in api itself since we get an undefined/null in the structure, which should not be the case at all, we should not work around it)

Have not seen it manifest however.

With all that, still unsure why this is in here...

Reply:
I double-checked api-observable, it has this - https://github.com/polkadot-js/api/blob/master/packages/api-observable/src/Queries.ts#L155

So it should never be null/undefined. If it is, something is going horribly wrong along the way.

Reply:
I just pulled and merged in the latest from 'next' branch, removed node_modules, restored the code back to `.add(fees.transferFee)`, and ran `yarn; yarn run build; yarn run check; yarn run start;` and this time no linting errors appeared. Although not sure what I did that triggered the linting error.